Adafruit_SHT31.h

```diff
@@ -62,21 +56,21 @@ class Adafruit_SHT31 {
    *
    * @return True if initialisation was successful, otherwise False.
    */
-  boolean begin(uint8_t i2caddr = SHT31_DEFAULT_ADDR);
+  bool begin(uint8_t i2caddr = SHT31_DEFAULT_ADDR);
 
   /**
    * Gets a single temperature reading from the sensor.
    *
    * @return A float value indicating the temperature.
    */
-  float readTemperature(void);
+  float readTemperature(void) { return readTempHum() ? temp : NAN; }
 
   /**
    * Gets a single relative humidity reading from the sensor.
    *
    * @return A float value representing relative humidity.
    */
-  float readHumidity(void);
+  float readHumidity(void) { return readTempHum() ? humidity : NAN; }
```

Comment:
no trinary - please revert this code to be in the cpp file

Reply:
done